arcgis-ios-sdk-samples/Geometry/Cut geometry/CutGeometry.storyboard

```diff
@@ -0,0 +1,62 @@
+    </device>
+    <dependencies>
+        <deployment identifier="iOS"/>
+        <plugIn identifier="com.apple.InterfaceBuilder.IBCocoaTouchPlugin" version="14088"/>
+        <capability name="Safe area layout guides" minToolsVersion="9.0"/>
+        <capability name="documents saved in the Xcode 8 format" minToolsVersion="8.0"/>
+    </dependencies>
+    <scenes>
+        <!--Cut Geometry View Controller-->
+        <scene sceneID="gFY-53-aKT">
+            <objects>
+                <viewController id="LQS-YS-sH4" customClass="CutGeometryViewController" customModule="arcgis_ios_sdk_samples" customModuleProvider="target" sceneMemberID="viewController">
+                    <view key="view" contentMode="scaleToFill" id="LVx-wq-HKJ">
+                        <rect key="frame" x="0.0" y="0.0" width="375" height="667"/>
+                        <autoresizingMask key="autoresizingMask" widthSizable="YES" heightSizable="YES"/>
+                        <subviews>
+                            <view contentMode="scaleToFill" translatesAutoresizingMaskIntoConstraints="NO" id="xCg-Rw-wvv" customClass="AGSMapView">
+                                <rect key="frame" x="0.0" y="20" width="375" height="603"/>
+                                <color key="backgroundColor" white="1" alpha="1" colorSpace="custom" customColorSpace="genericGamma22GrayColorSpace"/>
+                            </view>
+                            <toolbar opaque="NO" clearsContextBeforeDrawing="NO" contentMode="scaleToFill" translatesAutoresizingMaskIntoConstraints="NO" id="d9q-kU-Rfo">
+                                <rect key="frame" x="0.0" y="623" width="375" height="44"/>
+                                <constraints>
+                                    <constraint firstAttribute="height" constant="44" id="Cf2-Id-UiQ"/>
+                                </constraints>
+                                <items>
+                                    <barButtonItem style="plain" systemItem="flexibleSpace" id="d6H-db-A2S"/>
+                                    <barButtonItem title="Cut" id="OKg-tG-zkb">
+                                        <connections>
+                                            <action selector="cutGeometryWithPolyline:" destination="LQS-YS-sH4" id="yec-db-a1V"/>
+                                        </connections>
+                                    </barButtonItem>
+                                    <barButtonItem style="plain" systemItem="flexibleSpace" id="oT8-6Q-cii"/>
+                                </items>
+                            </toolbar>
+                        </subviews>
+                        <color key="backgroundColor" white="1" alpha="1" colorSpace="custom" customColorSpace="genericGamma22GrayColorSpace"/>
+                        <constraints>
+                            <constraint firstItem="xCg-Rw-wvv" firstAttribute="leading" secondItem="TTm-gO-6LF" secondAttribute="leading" id="9Hv-gA-rNU"/>
```

Comment:
Please constrain the map view and toolbar's leading and trailing edges to Superview rather than Safe Area.

![screen shot 2018-06-29 at 1 34 25 pm](https://user-images.githubusercontent.com/2257493/42113720-2efdc2dc-7ba1-11e8-96b0-70c6bfbf0c99.png)

Currently, the map view and toolbar don't use the full screen width on iPhone X in landscape:

![simulator screen shot - iphone x - 2018-06-29 at 13 35 48](https://user-images.githubusercontent.com/2257493/42113767-5d189a16-7ba1-11e8-9735-663e01dc4430.png)